Gemfile.lock

```diff
@@ -1,29 +1,26 @@
 GIT
   remote: git://github.com/LightGuard/awestruct.git
-  revision: 05bf2c7fc4027c00115b6bc2e9ff0ffcf3eacc8e
+  revision: 5dbe512680b7aa094598ecdcbb6e52d3e75aa32f
```

Comment:
Hrm, I didn't edit that on the gemfile.  I will add awestruct/awestruct to the Gemfile.